src/vstart.sh

```diff
@@ -1142,6 +1156,10 @@ start_osd() {
 	    # set exclusive CPU nodes for each osd
 	    echo "$CEPH_BIN/ceph -c $conf_fn config set osd.$osd crimson_seastar_cpu_cores $bottom_cpu-$top_cpu"
 	    $CEPH_BIN/ceph -c $conf_fn config set "osd.$osd" crimson_seastar_cpu_cores "$bottom_cpu-$top_cpu"
+        if [ $crimson_alien_num_threads -gt 0 ]; then
+	        echo "$CEPH_BIN/ceph -c $conf_fn config set osd.$osd crimson_alien_op_num_threads $crimson_alien_num_threads"
+	        $CEPH_BIN/ceph -c $conf_fn config set "osd.$osd" crimson_alien_op_num_threads "$crimson_alien_num_threads"
+        fi
```

Comment:
Since the value set here is the same for all Crimson OSDs it shouldn't be set for each OSD.
`crimson_alien_op_num_threads` is global to all of the OSDs:
```
$CEPH_BIN/ceph -c $conf_fn config set osd crimson_alien_op_num_threads "$crimson_alien_num_threads"
```

Reply:
Ah, thanks for clarifying, I was not aware of that 👍

Reply:
Can you please update the commit to use the global osd value?

Reply:
Sure, will do it asap